src/styles.css

```diff
@@ -14,7 +14,7 @@
   }
 
   .button-base {
-    @apply flex flex-row justify-center items-center rounded h-fit not-italic font-bold text-xl font-sans py-1;
+    @apply flex flex-row justify-center items-center rounded h-fit not-italic font-bold text-xl font-sans py-1 md:text-lg md:rounded-[10px] md:py-1;
```

Comment:
I'm wondering for the `rounded` attribute if we could use one of the Tailwind presets seen here: 
![image](https://user-images.githubusercontent.com/40529597/202942935-b1b2f98b-b1fb-4bb9-bb9e-e021ae6359fc.png) just bc this will make things more consistent!

Reply:
Also I just found out about this through Alton's PR but I think you can define multiple attributes for one breakpoint using brackets, so this could be:

`md: [text-lg py-1]`

Reply:
Also I noticed the buttons are less rounded on mobile versus desktop: 

![image](https://user-images.githubusercontent.com/40529597/202944262-e1235e67-ab4a-4953-804d-8026141dd550.png)
 
![image](https://user-images.githubusercontent.com/40529597/202944269-dc3f1251-24a7-44e7-bac5-1dff496f5c97.png)

Did we want that? I think it'd be better to keep the border-radius the same for both to be consistent!

Reply:
I just noticed but `md: [text-lg py-1]` actually doesn't work (it didn't work for Alton's PR either so we may have to go back and adjust that).

Reply:
Looks like it's not possible (see [this stackexchange](https://stackoverflow.com/questions/60567333/string-together-responsive-classes-in-tailwindcss)) :(